README.md

```diff
@@ -79,70 +94,69 @@ and build the project:
 
 ```shell
 $ dog-food -s smp.example.com:5223#KXNE1m2E1m0lm92WGKet9CL6+lO742Vy5G6nsrkvgs8=
 ```
 
 The base-64 encoded string in server address is the digest of RSA transport handshake key that the server will generate on the first run and output its digest.
 
-You can still talk to people using default or any other server, it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
+You can still talk to people using default or any other server; it only affects the location of the message queue when you initiate the connection (and the reply queue can be on another server, as set by the other party's client).
 
 Run `dog-food --help` to see all available options.
 
-## Using chat client
+### How to use SimpleX chat
 
-Once chat client is started, use `/add <name1>` to create a new connection and generate an invitation to send to your contact via any other communication channel (`<name1>` - is any name you want to use for that contact).
+This diagram shows how to connect and message a contact:
 
-Invitation has format `smp::<server>::<queue_id>::<rsa_public_key_for_this_queue_only>` - this needs to be shared with another party, via any other channel. This invitation can only be used once - even if this is intercepted, the attacker would not be able to use it to send you the messages via this queue once your contact confirms that the connection is established.
+<div align="center">
+  <img align="center" src="images/how-to-use-simplex.svg">
+</div>
 
-The party that received the invitation should use `/connect <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting party wants to use for you).
+Once you have started the chat, use `/add <name1>` to create a new connection and generate an invitation (`<name1>` - is any name you want to use for that contact). The add command will output an invitation. Send this invitation to your contact via any other channel.
 
-For example, if Alice and Bob want to chat, with Alice initiating, Alice would use [in her chat client]:
+The invitation has the format `smp::<server>::<queue_id>::<rsa_public_key_for_this_queue_only>`. The invitation can only be used once and even if this is intercepted, the attacker would not be able to use it to send you the messages via this queue once your contact confirms that the connection is established.
 
-```
-/add bob
-```
-
-And then send the generated invitation to Bob out-of-band. Bob then would use [in his chat client]:
+The contact who received the invitation should use `/connect <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting contact wants to use for you).
 
-```
-/connect alice <alice's invitation>
-```
+Once the contact has entered the `/connect` command, a connection is established and both parties are notified.
```

Comment:
maybe change `has entered` to `has executed`? it feels like `has entered` technically only implies `has typed`..